client/homebrew/pages/editPage/editPage.jsx

```diff
@@ -194,86 +260,134 @@ const EditPage = createClass({
 			htmlErrors : Markdown.validate(prevState.brew.text)
 		}));
 
-		const transfer = this.state.saveGoogle == _.isNil(this.state.brew.googleId);
+		if(this.editTypeIsNew()) {
```

Comment:
I feel like there should be some way we can better condense the saving logic between the New and Edit pages, since they both have to go through the hoops of whether to save to Google or not, etc. It would require some deeper digging though.

Reply:
Agreed - I've been pushing towards "functional" and less worried about "clean" or "elegant". But there's certainly space for it once the merging is complete.

Reply:
Hopefully the current commit is both cleaner and more elegant. I'll resolve this conversation and we can open new ones for anything that leaps out from the current commit.